meeting-notes/2020-06-01.md

```diff
@@ -0,0 +1,25 @@
+# Adeira working group meeting 01.06.2020
+
+## Attendees
+
+- Trond
+- Michal
+- Jarda
+
+## Can Relay compiler show warning when a deprecated field is used?
+
+It was possible but the API was changed and now it would require a lot of work. We can hope the compiler written in Rust will have a better API.
+
+Workaround - remove the deprecated field from a schema, run the compiler, check errors.
```

Comment:
> it would require a lot of work

I'd actually argue that now it's easier than it was before. 😎 But otherwise yeah, I am using exactly the workaround mentioned here when I want to be sure I cleaned the codebase properly. 🙂